tests/conditional_compilation/test_config.yml

```diff
@@ -1,15 +1,44 @@
 # Using models from https://github.com/openvinotoolkit/testdata
 # $find models -wholename "*.xml"
+# --------------
+# - model:
+#   path: <path_to_xml_file>
+#   libs:
+#       lib_name: [range] / range - the range of resizing the library in percent (old_size - new_size) / old_size * 100%
 
 - model:
     path: ${TESTDATA}/models/mobilenet_v2_1.4_224/mobilenet_v2_1.4_224_i8.xml
+    libs:
+        inference_engine_transformations: [82, 84]
```

Comment:
what does the array of numbers mean? `[82, 84]`
We should only have a name in a configuration